### DISCUSSION TITLE
Startup failure after reboot

### DISCUSSION THREAD

**Discussion by ghost** [postalserver/postal#1606] [Help with using Postal]
Have postal running on fresh Ubuntu 20.04. Server rebooted automatically this AM after auto security update and Postal failed to start. Looking at the logs it seems to shutdown after timing out to the database. What would be the best way to approach this issue? Handled via Compose or would this be considered an app bug?

 /usr/local/bundle/gems/mysql2-0.5.3/lib/mysql2/client.rb:131:in `_query': Lost connection to MySQL server during query (Mysql2::Error::ConnectionError)
worker_1    | 	from /usr/local/bundle/gems/mysql2-0.5.3/lib/mysql2/client.rb:131:in `block in query'
worker_1    | 	from /usr/local/bundle/gems/mysql2-0.5.3/lib/mysql2/client.rb:130:in `handle_interrupt'
worker_1    | 	from /usr/local/bundle/gems/mysql2-0.5.3/lib/mysql2/client.rb:130:in `query'
worker_1    | 	from /usr/local/bundle/gems/activerecord-5.2.6/lib/active_record/connection_adapters/abstract_mysql_adapter.rb:187:in `block (2 levels) in execute'
worker_1    | 	from /usr/local/bundle/gems/activesupport-5.2.6/lib/active_support/dependencies/interlock.rb:48:in `block in permit_concurrent_loads'
worker_1    | 	from /usr/local/bundle/gems/activesupport-5.2.6/lib/active_support/concurrency/share_lock.rb:187:in `yield_shares'
worker_1    | 	from /usr/local/bundle/gems/activesupport-5.2.6/lib/active_support/dependencies/interlock.rb:47:in `permit_concurrent_loads'
worker_1    | 	from /usr/local/bundle/gems/activerecord-5.2.6/lib/active_record/connection_adapters/abstract_mysql_adapter.rb:186:in `bloc

Thanks!

**Comment by willpower232** [comment]
How are you running mysql on your server? you may be interested in #1562

**Reply by ghost** [reply]
Running everything in docker as per install instructions. Thanks for the reference but there's nothing specific in that thread in how to solve?

**Reply by willpower232** [reply]
You can see that the problem is Postal cannot talk to your database. If you have just rebooted then MySQL will not be running yet as both it and RabbitMQ are slower to start than Postal is so you need to ask Postal to wait using the configuration suggested in that thread.
